XMonad/Util/ClickableWorkspaces.hs

```diff
@@ -47,16 +46,15 @@ xmobarEscape = concatMap doubleLts
 
 clickableWrap :: Integer -> String -> String
 clickableWrap num ws =
-  "<action=xdotool key super+" ++ show num ++ ">" ++ xmobarEscape ws ++ "</action>"
+  "<action=wmctrl -s " ++ show num ++ ">" ++ xmobarEscape ws ++ "</action>"
```

Comment:
Consider using [`xmobarRaw`](https://hackage.haskell.org/package/xmonad-contrib-0.16/docs/XMonad-Hooks-DynamicLog.html#v:xmobarRaw) instead of your own `xmobarEscape`. I think xmobar doesn't unescape the double `<<` back to a single one, does it?

Reply:
Oh and you might want to use https://hackage.haskell.org/package/xmonad-contrib-0.16/docs/XMonad-Hooks-DynamicLog.html#v:xmobarAction as well :-)

Reply:
This actually reminded me that I stupidly never got around to testing whether the escape was relevant! I just wrongly assumed that it was relevant (maybe it used to be, when the snippet I saw it in was written?), but from testing it just now, it simply makes double the number of `<`s show up in the bar. I never have had them in my workspace names and forgot to check what would happen if I did Thank you for pointing it out.

New commit is using both of those utility functions, much cleaner (and more correct to boot).

Reply:
Well, it's unlikely that someone would have a malicious action in their workspace name, but it's easier to just use the correct escape and not worry about it. :-)

Way more important is correct escaping of window titles, which is already handled in the default pp: https://github.com/xmonad/xmonad-contrib/blob/3dc49721b69f5c69c5d5e1ca21083892de72715d/XMonad/Hooks/DynamicLog.hs#L549
Not sure I like this default implementation (I think `xmobarRaw` should be the default for `xmobarPP` these days), but that's totally off-topic here.

Reply:
Turns out if someone wants to place icons in their workspace names, this escaping stands in the way. So I think I'll drop it: https://github.com/liskin/xmonad-contrib/commit/283636e29e3f0978c7418664004e27f76bbdfc6e